tests/TestData/Providers/Reflection/ReflectionFunctionsProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace StubTests\TestData\Providers\Reflection;
+
+use Generator;
+use StubTests\Model\StubProblemType;
+use StubTests\TestData\Providers\EntitiesFilter;
+use StubTests\TestData\Providers\ReflectionStubsSingleton;
+
+class ReflectionFunctionsProvider
+{
+    public static function allFunctionsProvider(): ?Generator
+    {
+        foreach (EntitiesFilter::getFiltered(ReflectionStubsSingleton::getReflectionStubs()->getFunctions()) as $function) {
```

Comment:
Can we omit `EntitiesFilter::getFiltered` call?

Reply:
The same answer as above. `getFiltered` by default filters entities with known "STUBS_IS_MISSED"